content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,215 +1,157 @@
 ---
 title: 'Is Prisma an ORM?'
 metaTitle: 'Is Prisma an ORM?'
-metaDescription: 'Learn why Prisma is not an ORM. It shares similar goals with ORMs and wants to make working with databases easy, but it does not map classes to tables as ORMs do.'
+metaDescription: 'Learn about how Prisma implements the Data Mapper ORM pattern and how it achieves the same goal as traditional ORMs without requiring you to map classes to tables as traditional ORMs do.'
 ---
 
-<TopBlock>
+## Overview
```

Comment:
Is it normal that we use "Overview" now instead of the TopBlock?

Reply:
Changed to TopBlock.